spec/dummy/config/master.key

```diff
@@ -0,0 +1 @@
+363a97a4bd022964407e1ac663b1a20e
```

Comment:
should the master key be here? 🤔

Reply:
@soni-sahu Yup -- it's fine. This is just a dummy app for testing. It's not an actual web app that will be served anywhere.